diff --git setup.py setup.py
index 55ecc2f..b018cf0 100755
--- setup.py
+++ setup.py
@@ -97,35 +97,35 @@ sys.path.pop(-1)
 class pivy_build(build):
     PIVY_SNAKES = r"""
                             _____
-                        .-'`     '.                                     
-                     __/  __       \                                   
-                    /  \ /  \       |    ___                          
-                   | /`\| /`\|      | .-'  /^\/^\                   
-                   | \(/| \(/|      |/     |) |)|                     
-                  .-\__/ \__/       |      \_/\_/__..._             
-          _...---'-.                /   _              '.               
-         /,      ,             \   '|  `\                \           
-        | ))     ))           /`|   \    `.       /)  /) |             
-        | `      `          .'       |     `-._         /               
-        \                 .'         |     ,_  `--....-'               
-         `.           __.' ,         |     / /`'''`                     
-           `'-.____.-' /  /,         |    / /                           
-               `. `-.-` .'  \        /   / |                           
-                 `-.__.'|    \      |   |  |-.                         
-                    _.._|     |     /   |  |  `'.                       
-              .-''``    |     |     |   /  |     `-.                    
-           .'`         /      /     /  |   |        '.                  
-         /`           /      /     |   /   |\         \               
-        /            |      |      |   |   /\          |               
-       ||            |      /      |   /     '.        |                
-       |\            \      |      /   |       '.      /              
-       \ `.           '.    /      |    \        '---'/               
-        \  '.           `-./        \    '.          /                
-         '.  `'.            `-._     '.__  '-._____.'--'''''--.         
-           '-.  `'--._          `.__     `';----`              \       
-              `-.     `-.          `.''```                     ;        
-                 `'-..,_ `-.         `'-.                     /         
-                        '.  '.           '.                 .'          
+                        .-'`     '.
+                     __/  __       \
+                    /  \ /  \       |    ___
+                   | /`\| /`\|      | .-'  /^\/^\
+                   | \(/| \(/|      |/     |) |)|
+                  .-\__/ \__/       |      \_/\_/__..._
+          _...---'-.                /   _              '.
+         /,      ,             \   '|  `\                \
+        | ))     ))           /`|   \    `.       /)  /) |
+        | `      `          .'       |     `-._         /
+        \                 .'         |     ,_  `--....-'
+         `.           __.' ,         |     / /`'''`
+           `'-.____.-' /  /,         |    / /
+               `. `-.-` .'  \        /   / |
+                 `-.__.'|    \      |   |  |-.
+                    _.._|     |     /   |  |  `'.
+              .-''``    |     |     |   /  |     `-.
+           .'`         /      /     /  |   |        '.
+         /`           /      /     |   /   |\         \
+        /            |      |      |   |   /\          |
+       ||            |      /      |   /     '.        |
+       |\            \      |      /   |       '.      /
+       \ `.           '.    /      |    \        '---'/
+        \  '.           `-./        \    '.          /
+         '.  `'.            `-._     '.__  '-._____.'--'''''--.
+           '-.  `'--._          `.__     `';----`              \
+              `-.     `-.          `.''```                     ;
+                 `'-..,_ `-.         `'-.                     /
+                        '.  '.           '.                 .'
 
 
                             ~~~ HISSSSSSSSSS ~~~
@@ -234,7 +234,7 @@ class pivy_build(build):
         else:
             try:
                 import qtinfo
-                self.QTINFO = qtinfo.QtInfo()
+                self.QTINFO = qtinfo.QtInfo(qmake_command=[%QMAKE_COMMAND%])
             except Exception as e:
                 import traceback
                 print(red("\ndisable soqt, because there was a problem running qtinfo (needs qmake)"))
@@ -551,11 +551,11 @@ class pivy_build(build):
                     _INCLUDE_DIR = INCLUDE_DIR.replace('"', "")
                 CPP_FLAGS += ' -I' + _INCLUDE_DIR
                 CPP_FLAGS += ' -I' + os.path.join(_INCLUDE_DIR, 'Inventor', 'annex')
-                if sys.platform == 'win32': 
+                if sys.platform == 'win32':
                     CPP_FLAGS += " /DCOIN_DLL /wd4244 /wd4049"
                     LDFLAGS_LIBS = quote(max(glob.glob(os.path.join(LIB_DIR, "Coin?.lib")))) + " "
                 else:
-                    CPP_FLAGS += " -Wno-unused -Wno-maybe-uninitialized"
+                    CPP_FLAGS += " -Wno-unused -Wno-sometimes-uninitialized -std=c++11"
                     LDFLAGS_LIBS = ' -L' + self.cmake_config_dict[config_cmd + '_LIB_DIR']
 
                 if module == "soqt":
@@ -569,7 +569,7 @@ class pivy_build(build):
                         CPP_FLAGS += " /DSOQT_DLL"
                     else:
                         LDFLAGS_LIBS += ' -lSoQt'
-                
+
                 if module == "coin":
                     if sys.platform == 'win32':
                         pass
@@ -665,7 +665,7 @@ for i in reversed(list(range(len(sys.argv)))):
         pivy_build.MODULES.pop(sys.argv[i][10:], None)
         del sys.argv[i]
 
- 
+
 setup(name="Pivy",
       version=PIVY_VERSION,
       description="A Python binding for Coin",
